docs/src/QuickStart/building-with-cmake.md

```diff
@@ -0,0 +1,88 @@
+# Building with CMake
+
+## Prerequisites
+
+It is necessary for you to have the following tools before you get started on
+making a new build from this project:
+
+- C++ compiler
+- CMake
+- LLVM 13
+
+If you are compiling and using the command-line tool, these are necessary tools
+regardless of whether you plan to develop the library itself. The build
+configuration is handled by CMake, which informs you if a compiler needs to be
+installed.
+
+On all systems, you will additionally need to check the submodules out. From the
+root of the repository, run
+
+```sh
+git submodule update --init --recursive
+```
+
+### Prerequisites on Windows
+
+Currently CMake builds are not supported on Windows using PowerShell. It is
```

Comment:
No, actually, the problem is LLVM. I spent around a week with Ian in the beginning of the year to get everything compiling with LLVM. The issue is to find a precompiled version of LLVM that is easily installable and afterwards findable by CMake. We could fix this problem by making CMake rely on building LLVM from source, but I don't think that would be beneficial since currently the biggest benefit of LLVM is that it can use system libaries.

Reply:
If you are using WSL on the other hand, it is my understanding that everything works like it does on Ubuntu, but I have not tested it.